src/lib/backend.ai-client-esm.ts

```diff
@@ -213,9 +213,9 @@ class Client {
   static ERR_UNKNOWN: any;
 
   // Additional info related to current login user
-  public email: string;
-  public full_name: string;
-  public user_uuid: string;
+  public email!: string;
+  public full_name!: string;
+  public user_uuid!: string;
```

Comment:
If it's not being used, isn't it okay to delete it?

Reply:
I got `Property 'email'/'full_name'/'user_uuid' has no initializer and is not definitely assigned in the constructor.` warning message. So, I simply added it.